.circleci/config.yml

```diff
@@ -80,6 +80,18 @@ workflows:
   build:
     jobs:
       - test:
+          filters:
+            branches:
+              # separate `build-and-notify` workflow for these branches includes sending notifications
+              ignore: ['main', 'develop']
+          context:
+            - docker-hub-creds
+  build-and-notify:
+    jobs:
+      - test:
+          filters:
+            branches:
+              only: ['main', 'develop']
```

Comment:
I don't think Circle allows filtering the post-steps specifically, so had to duplicate the workflow.

If we're happy with this I'll do the same thing in matchbot at https://github.com/thebiggive/matchbot/blob/77933d51dcc66e24875f3612c2154ce1b448b8cb/.circleci/config.yml#L139

Reply:
This is one reason I mostly prefer CLI tools over Circle orbs - if we were using a CLI tool for slack directly then instead of duplicating the workflow here I'd have put code on the command line to say only notify slack if branch is main or develop.